test/Project.toml

```diff
@@ -14,6 +14,6 @@ UnfoldMakie = "69a5ce3b-64fb-4f22-ae69-36dd4416af2a"
 UnfoldSim = "ed8ae6d2-84d3-44c6-ab46-0baf21700804"
 
 [compat]
```

Comment:
Please remove these compat entries entirely² - then the package will use the compat entries of UnfoldMakie/Project.toml.

we want to test after all, if it works with newest versions of everything

² in theory your tests could depend on non-UnfoldMakie dependencies of older versions, but this is not good practice, as sub-dependencies then could be different for users and for the tests